app/templates/event/partials/edit_event_details_tab.html

```diff
@@ -40,15 +59,37 @@
     </select>
   </div>
 
+  <div class="form_row_end">
+      <select name="availability">
+          <option value="False">Free</option>
+          <option value="True" selected>Busy</option>
+      </select>
+      <select name="privacy">
+          <option value="private">Private</option>
+          <option value="public" selected>Public</option>
+      </select>
+  </div>
+</div>
 
-    <div class="form_row_end">
-        <select name="availability">
-            <option value="False">Free</option>
-            <option value="True" selected>Busy</option>
-        </select>
-        <select name="privacy">
-            <option value="private">Private</option>
-            <option value="public" selected>Public</option>
-        </select>
-    </div>
-</div>
\ No newline at end of file
+<script>
+  var items_num = 1;
+
+  function addItem() {
+    var item = document.getElementById("Item").cloneNode(true);
+    item.style.display = "flex";
+    item.childNodes[7].id += String(items_num);
```

Comment:
Don't use magic numbers in your code

Reply:
Although my updated code is now using variables instead of magic numbers, but I couldn't understand how to access childNodes by ID without using numbers. Any ideas? The current updated code is fine?